recipes-kernel/linux/linux-arc.inc

```diff
@@ -1,29 +1,24 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/linux-arc:"
 DESCRIPTION = "Kernel image for the ARC architecture"
 SECTION = "kernel"
 
 LICENSE = "GPLv2"
 
 inherit kernel
 
-SRC_URI = "\
-    https://www.kernel.org/pub/linux/kernel/v4.x/linux-${PV}.tar.xz\
-    file://0009-UDL-add-pseudo-deferred-io-for-modeset-drive.patch \
-    file://0010-UDL-defio-add-fb-lock-and-required-checks.patch \
-    file://0011-ARC-hsdk-Enable-U-Boot-support.patch \
-    file://0012-ARC-hsdk-Enable-cgroups-support.patch \
```

Comment:
Note we still don't have `CGROUPS` added in `hsdk_defconfig` at least in 5.3.y, see https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/arch/arc/configs/hsdk_defconfig?h=linux-5.3.y so is that really OK to get rid of this patch here and now?

I guess if we use Yocto kernel then we'd be able to add cgroups support via config fragment, see http://git.yoctoproject.org/cgit/cgit.cgi/yocto-kernel-cache/tree/features/cgroups.

So maybe if we build custom kernel this is indeed OK to not mess with anything except pristine defconfig... well if kernel boots obviously.